### PR TITLE
Multi scan

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: xpdacq/tests/test_xrun.py [hunk 1]
```diff
@@ -422,6 +422,15 @@ def test_facility_md(self):
...
 
+    def test_double_scan(self):
+        key_list = ['owner', 'facility', 'group']
+        for k in key_list:
+            self.xrun.md[k] = glbl[k]
+        self.xrun({'sample_name': 'double_scan'},
+                  [ScanPlan(self.bt, ct, 1.0), ScanPlan(self.bt, ct, 1.0)])
+        for i in range(1, 3):
+            h = xpd_configuration['db'][-1 * i]
+            assert h['start']['sample_name'] == 'double_scan'
```
Comment (chiahaoliu) [inline xpdacq/tests/test_xrun.py]: just a small point, can we also validate the ScanPlan md contents?

File: xpdacq/xpdacq.py [hunk 2]
```diff
@@ -473,42 +614,37 @@ def __call__(self, sample, plan, subs=None, *,
...
-                return
-        # If the plan is an xpdAcq 'ScanPlan', make the actual plan.
-        if isinstance(plan, ScanPlan):
-            plan = plan.factory()
+        # Turn everything into lists
+        sample, plan = self._normalize_sample_plan(sample, plan)
+        # Turn ints into actual samples
+        sample = self.translate_to_sample(sample)
+        # Turn ints into generators
+        plan = self.translate_to_plan(plan, sample)
```
Comment (chiahaoliu) [inline xpdacq/xpdacq.py]: 👍

**Comment by CJ-Wright** [comment]
Ready for more review/merge

**Comment by chiahaoliu** [comment]
LGTM

**Pull request merged**